### ISSUE TITLE
[Bug]: double uploads

### ISSUE THREAD

**Issue by subbass1** [commons-app/apps-android-commons#5759]
### Summary

I quite often suffer from double uploads. E.g. when I upload the nett files the bext day, somd of the old files were uploaded again

 

### Steps to reproduce

Nothing special

### Expected behaviour

..........

### Actual behaviour

Double uploads

### Device name

galaxy s23 ultra

### Android version

Android 14

### Commons app version

5.0.1~da0b2c28e

### Would you like to work on the issue?

None

**Comment by sivaraam** [comment]
Hi @subbass1 ! Thabk you for opening the issue. It would be helpful if you could elaborate on wjat specific case you observed this issue. It seems to me like you had some queued uploads which you resumed the next day and faced issues due to it. Is that it?

Also, which images were uploaded twice? The images that were already uploaded or the ones you were being uploaded at that point?

**Comment by mnalis** [comment]
@subbass1 has the issue been fixed for you with recent releases (5.1.2)? 
There have been quite a few changes to upload code, so unless this  can be reproduced, this issue could probably be closed?

**Comment by subbass1** [comment]
Yes, fixed, does not happen anymore. Also quite an amount of multiple uploads (say 50) are working without hassles, years ago that was not possible.

**Comment by mnalis** [comment]
That is great news @subbass1! Could you then please click on "Close issue" button so it is marked as resolved? thanks!
